README.md

```diff
@@ -8,8 +8,34 @@ Level Up is a Steam game recommendations site where users can submit their own a
 
 ## Run it yourself
 
-Level Up's server components are written in Go and operate on top of an Amazon DynamoDB database (DynamoDB's free tier is more than enough to host).
+### 1. using Docker
+
+Make sure you have the .env file if not run: `cp .env.example .env && $EDITOR .env`
+```
+`docker build -t levelup .` 
```

Comment:
Can you please remove the opening backticks here and in the other code blocks? When rendered they show literally, which I don't think was intended.